opmd_viewer/openpmd_timeseries/plotter.py

```diff
@@ -354,20 +354,14 @@ def show_field_2d(self, F, info, slicing_dir, m, field_label, geometry,
         # Cylindrical geometry
         if geometry == "thetaMode":
             mode = str(m)
-            plt.title("%s in the mode %s at %.1f s   (iteration %d)"
+            plt.title("%s in the mode %s at %.2e s   (iteration %d)"
                       % (field_label, mode, time, iteration),
                       fontsize=self.fontsize)
         # 2D Cartesian geometry
-        elif geometry == "2dcartesian":
-            plt.title("%s at %.1f s   (iteration %d)"
+        else:
+            plt.title("%s at %.2e s   (iteration %d)"
                       % (field_label, time, iteration),
                       fontsize=self.fontsize)
-        # 3D Cartesian geometry
-        elif geometry == "3dcartesian":
-            slice_plane = info.axes[0] + '-' + info.axes[1]
-            plt.title("%s sliced in %s at %.1f s  (iteration %d)"
-                      % (field_label, slice_plane, time, iteration),
-                      fontsize=self.fontsize)
```

Comment:
I also suppressed the slicing information that was specific for the 3D case, since 2D and circ data can now also be sliced.